variables.env

```diff
@@ -0,0 +1 @@
+RABBIT_URI=amqp://rabbit
```

Comment:
By doing this, do we keep the default `?timeout=30`? If not, we should parse the URL and auto append query params to make it 30s if not otherwise defined.